migration/tasks.py

```diff
@@ -190,6 +191,10 @@ def update_collection(collection, query, doc):
 
 
 def update_doc(collection, document):
+    # update place items with geojson
+    if collection.name = 'places':
+        get_geojson(document)
```

Comment:
get_geojson does not change `document`. you have to add it's return value to the document